Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -448,6 +448,14 @@ public override void Update(float elapsedClockSeconds)
                     AuxResPressurePSI += dp;
                     BrakeLine1PressurePSI -= dp * AuxBrakeLineVolumeRatio;  // Adjust the train brake pipe pressure
                 }
+                if (AuxResPressurePSI > BrakeLine1PressurePSI) // Allow small flow from auxiliary reservoir to brake pipe to restore normal pressure after overcharge
+                {
+                    float dp = elapsedClockSeconds * 0.05f;
```

Comment:
Please explain the justification for the number `0.05f` and move it to a constant with a descriptive name.